business/workloads.go

```diff
@@ -746,7 +746,7 @@ func fetchWorkloads(ctx context.Context, layer *Layer, namespace string, labelSe
 	for _, pod := range pods {
 		if len(pod.OwnerReferences) != 0 {
 			for _, ref := range pod.OwnerReferences {
-				if ref.Controller != nil && *ref.Controller {
+				if ref.Controller != nil && *ref.Controller && !isWorkloadIncluded(ref.Kind) {
```

Comment:
This should be the opposite:
```suggestion
				if ref.Controller != nil && *ref.Controller && isWorkloadIncluded(ref.Kind) {
```